### PR TITLE
Remade dashboard version - not done

### PR DESCRIPTION
**Pull request by Niputi**
Ref: https://github.com/Mikibot/sprint-tasks/issues/20

* dashboard ui
* auth

### CODE REVIEW

File: components/nav.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Link from 'next/link'
+
+const links = [
+  { href: 'https://zeit.co/now', label: 'ZEIT' },
+  { href: 'https://github.com/zeit/next.js', label: 'GitHub' }
```
Comment (velddev) [inline components/nav.js]: Pretty sure this isn't related to Miki?

File: components/nav.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <li key={key}>
+          <a href={href}>{label}</a>
+        </li>
+      ))}
+    </ul>
+
+    <style jsx>{`
+      :global(body) {
+        margin: 0;
+        font-family: -apple-system, BlinkMacSystemFont, Avenir Next, Avenir,
```
Comment (velddev) [inline components/nav.js]: We use Roboto, Arial as backup

File: package.json [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
-		"build": "concurrently \"npm run build:server\" \"npm run build:web\"",
-		"build:server": "cd server && npm install",
-		"build:web": "cd web && npm install",
-		"start:server": "cd server && npm start",
-		"start:web": "cd web && npm start"
-	},
-	"dependencies": {
-		"concurrently": "^4.1.0"
-	}
+  "name": "my-app",
```
Comment (velddev) [inline package.json]: miki-dashboard

File: README.md [hunk 4]
```diff
@@ -1,17 +1 @@
...
-2. docker run miki-dashboard -p 8080:8080
-
-### The non-docker way (development)
-1. `npm install`
-2. `npm run build`
-4. `npm start`
-
-The docker build is designed to work on heroku in specific and doesn't have
-the hot-reloading you get with `npm start`
+# Miki.Web.Dashboard
```
Comment (velddev) [inline README.md]: Atleast write something here please. There's a great example for default readmes in [the dotnet template](https://github.com/mikibot/template-dotnet-project)

**Comment by Niputi** [comment]
merging because veld wanted this in master

**Pull request merged**